Maya: Passing custom attributes to alembic - OP-4111 (#4516)
ynput/OpenPype: website/docs/artist_hosts_maya.md

![Maya - Point Cache Example](assets/maya-pointcache_setup.png)

#### Options

- **Frame Start**: which frame to start the export at.
- **Frame End**: which frame to end the export at.
- **Handle Start**: additional frames to export at frame start. Ei. frame start - handle start = export start.
- **Handle Start**: additional frames to export at frame end. Ei. frame end + handle end = export end.
- **Step**: frequency of sampling the export. For example when dealing with quick movements for motion blur, a step size of less than 1 might be better.
- **Refresh**: refresh the viewport when exporting the pointcache. For performance is best to leave off, but certain situations can require to refresh the viewport, for example using the Bullet plugin.

> - **Attr**: specific attributes to publish separated by `;`

Reviewer: There's also `attrPrefix` by the way.

Anyway, the new publisher #4388 exposes both of these nicely in the UI.

Author: ```suggestion
- **Attr**: specific attributes to publish separated by `;`.
- **AttrPrefix**: Prefix filter for determining which geometric attributes to write out, separated by `;`.
```

- **Handle Start**: additional frames to export at frame end. Ei. frame end + handle end = export end.
- **Step**: frequency of sampling the export. For example when dealing with quick movements for motion blur, a step size of less than 1 might be better.
- **Refresh**: refresh the viewport when exporting the pointcache. For performance is best to leave off, but certain situations can require to refresh the viewport, for example using the Bullet plugin.

> - **Attr**: specific attributes to publish separated by `;`.
> - **AttrPrefix**: Prefix filter for determining which geometric attributes to write out, separated by `;`.

Reviewer: Sorry to be precise, but it's not only geometric - it'd be any attribute that matches it, also on transforms.

```suggestion
- **Attr**: specific attributes to publish separated by `;`.
- **AttrPrefix**: specific attributes which start with this prefix to publish separated by `;`.
```

Or:
```suggestion
- **Attr**: include attributes with these names in the publish separated by `;`, for example `my_attribute;other_attribute`
- **AttrPrefix**: include attributes starting with this prefix in the publish separated by `;`, for example `ai;vray`
```
